jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -216,6 +216,7 @@ public final void run() {
 	 */
 	public static boolean yield(ContinuationScope scope) {
 		/* TODO find matching scope to yield */
+		Continuation cont = JLA.getContinuation(JLA.currentCarrierThread());
```

Comment:
Improves readability and eases debugging when investigating failures.
```suggestion
		Thread carrierThread = JLA.currentCarrierThread();
		Continuation cont = JLA.getContinuation(carrierThread);
```

Reply:
fixed